packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -44,57 +44,204 @@ const isDropdownSelect = (element = {}) => {
+   */
     highlightDates: PropTypes.instanceOf(Map),
+    /**
+   * Show dates only in the given array.
+   */
     includeDates: PropTypes.array,
+    /**
+   * Prop to show inline version of date picker component.
+   */
     inline: PropTypes.bool,
+    /**
+   * Name of locale data for different international formatting.
+   */
     locale: PropTypes.string,
+    /**
+   * Maximum Value of date that can be selected by user.
+   */
     maxDate: PropTypes.object,
+    /**
+   * Minimum Value of date that can be selected by user.
+   */
     minDate: PropTypes.object,
+    /**
+   * Prop to show multiple months on date picker.
+   */
     monthsShown: PropTypes.number,
+    /**
+   * A callback function that is executed when user clicks outside the datepicker.
+   */
     onClickOutside: PropTypes.func,
+    /**
+   * A callback function that is executed when month is selected.
+   */
     onMonthChange: PropTypes.func,
+    /**
+   * Prop to show month navigation.
+   */
     forceShowMonthNavigation: PropTypes.bool,
+    /**
+   * A callback function that is executed when date picker is clicked for dropdown.
+   */
     onDropdownFocus: PropTypes.func,
+    /**
+   * A callback function that is executed when a valid date is selected.
+   */
     onSelect: PropTypes.func.isRequired,
+    /**
+   * A callback function that is executed when a Week number is selected.
+   */
     onWeekSelect: PropTypes.func,
+    /**
+   * A boolean prop to show time picker.
+   */
     showTimeSelect: PropTypes.bool,
+    /**
+   * Prop to choose format of time selected.
+   */
     timeFormat: PropTypes.string,
+    /**
+   * Interval between 2 consecutive times on time picker in minutes.
+   */
     timeIntervals: PropTypes.number,
+    /**
+   * A callback function to execute when a valid time is selected.
+   */
     onTimeChange: PropTypes.func,
+    /**
+   * Minimum Value of time that can be selected by user.
+   */
     minTime: PropTypes.object,
+    /**
+   * Maximum Value of time that can be selected by user.
+   */
     maxTime: PropTypes.object,
+    /**
+   * Array to store time values disabled for selection.
+   */
     excludeTimes: PropTypes.array,
+    /**
+   * Prop to open calendar on the given date.
+   */
     openToDate: PropTypes.object,
+    /**
+   * Prop to show dates of next month also.
+   */
     peekNextMonth: PropTypes.bool,
+    /**
+   * Prop to show a scrollable dropdown to choose year on the date picker.
+   */
     scrollableYearDropdown: PropTypes.bool,
+    /**
+   * Prop to store previous selection.
+   */
     preSelection: PropTypes.object,
+    /**
+   * Selected Date Value.
+   */
     selected: PropTypes.object,
+    /**
+   * Mark date picker  to select end of range .
+   */
     selectsEnd: PropTypes.bool,
+    /**
+   * Mark date picker  to select start of range .
+   */
     selectsStart: PropTypes.bool,
+    /**
+   * Prop to show a dropsown to select month in date picker .
```

Comment:
Small typo for `dropdown`

If you are using VS Code as your editor there is a spell checker plugin that I have found to be really useful.

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
Resolved in 2631f0f